package com.chargify.core.resources;

import org.junit.Test;

import static org.junit.Assert.*;

public class CustomerTest {

    @Test
    public void testFind() throws Exception {
    }
}